pages/editor.vue

```diff
@@ -67,31 +86,72 @@ export default {
   },
   data() {
     return {
-      markType: 'voice',
+      markType: 'quote',
```

Comment:
I prefer constants to be written in caps, so that you dont confuse them with normal string content

Reply:
Actually, that’s not a constant, but is used in order to show the correct modal. I don’t really like this either, but I won’t fix it for know – sorry! :)